architecture/nvidia-gpu-architecture-overview.adoc

```diff
@@ -0,0 +1,68 @@
+
+include::modules/nvidia-gpu-virtualization.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/openshift-virtualization.html[NVIDIA GPU Operator with OpenShift Virtualization]
+
+include::modules/nvidia-gpu-vsphere.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/nvaie-with-ocp.html#openshift-container-platform-on-vmware-vsphere-with-nvidia-vgpus[OpenShift Container Platform on VMware vSphere with NVIDIA vGPUs]
+
+include::modules/nvidia-gpu-kvm.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://computingforgeeks.com/how-to-deploy-openshift-container-platform-on-kvm/[How To Deploy OpenShift Container Platform 4.13 on KVM]
+
+include::modules/nvidia-gpu-csps.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/ai-enterprise/deployment-guide-cloud/0.1.0/aws-redhat-openshift.html[Red Hat Openshift in the Cloud]
+* link:https://access.redhat.com/solutions/6177662[OpenShift 4 and GPU requirements in Google Cloud Platform (GCP)]
+
+include::modules/nvidia-gpu-red-hat-device-edge.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://cloud.redhat.com/blog/how-to-accelerate-workloads-with-nvidia-gpus-on-red-hat-device-edge[How to accelerate workloads with NVIDIA GPUs on Red Hat Device Edge]
+
+include::modules/nvidia-gpu-features.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.nvidia.com/ngc/ngc-deploy-on-premises/nvidia-certified-systems/index.html[NVIDIA-Certified Systems]
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/nvidia-gpu-admin-dashboard#doc-wrapper[The NVIDIA GPU administration dashboard]
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/nvaie-with-ocp.html[NVIDIA AI Enterprise with OpenShift]
+* link:https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/overview.html#[NVIDIA Container Toolkit]
+* link:https://developer.nvidia.com/dcgm[NVIDIA DCGM]
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/mig-ocp.html[MIG Support in OpenShift Container Platform]
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/time-slicing-gpus-in-openshift.html[Time-slicing NVIDIA GPUs in OpenShift]
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/mirror-gpu-ocp-disconnected.html[Deploy GPU Operators in a disconnected or airgapped environment]
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/install-nfd.html[Installing the Node Feature Discovery (NFD) Operator]
```

Comment:
Is there a reason to to skip the [OCP NFD Operator install instructions](https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html#installing-the-node-feature-discovery-operator_node-feature-discovery-operator) for Nvidia?